packages/cli/build.gradle.kts

```diff
@@ -1102,6 +1108,9 @@ val commonNativeArgs = listOfNotNull(
   "-H:+PreserveFramePointer",
   "-H:+ReportExceptionStackTraces",
   "-H:+AddAllCharsets",
+  "-H:-IncludeLanguageResources",
+  "-H:+CopyLanguageResources",
```

Comment:
fixes issue with embedded python resources. `python-home` is no longer embedded within the image, and is maintained in a sidecar tree of files next to the binary. in jvm mode, we let GraalVM configure paths.